CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ option(QUIC_BUILD_TOOLS "Builds the tools code" ON)
 option(QUIC_BUILD_TEST "Builds the test code" ON)
 option(QUIC_BUILD_PERF "Builds the perf code" ON)
 option(QUIC_ENABLE_LOGGING "Enables logging" ON)
-option(QUIC_SANITIZE_ADDRESS "Enables address sanitizer" OFF)
+option(QUIC_ENABLE_SANITIZERS "Enables sanitizers" OFF)
```

Comment:
This conflicts a bit with #606, which I should eventually add :)

Maybe QUIC_ENABLE_SANITIZER=thread, QUIC_ENABLE_SANITIZER=address?

Reply:
I'm in favor of merging now and letting you change things once #606 is ready :-)